content/learn/contribute/helping-out/opening-pull-requests.md

```diff
@@ -147,3 +149,9 @@ When the new pull request is ready, it should reference the original PR in the d
 Then, notify org members to close the original.
 
 * For example, you can reference the original PR by adding the following to your PR description: `Adopted #number-original-pull-request`
+
+## Helping a PR get ready
+
+Without going to the complete adoption of a PR, sometimes the author needs help to get it approved or passing CI.
+Those PRs can be labeled as *[S-Needs-Help](https://github.com/bevyengine/bevy/labels/S-Needs-Help)*, and opening PRs on them is welcomed to fix the last few points, resolve conflicts to pass CI.
+You will need to work closely with the original author or one of the maintainer to add your commits to the original PR.
```

Comment:
```suggestion
You will need to work closely with the original author or one of the maintainer to add your commits to the original PR. This "needs help" process should focus on newer PRs for the most part. Lean toward closing PRs which are more than six months old, adding the *[S-Adopt-Me](https://github.com/bevyengine/bevy/labels/S-Adopt-Me)* label.
```
I wonder if it's worth adding something like this to help reinforce the idea of not hanging on to stale PRs indefinitely?